src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
```

Comment:
The class FileServiceImpl should not start with an empty line. Please remove the empty line at the beginning of the class definition.